canonical-web-and-design/snapcraft.io
# Mind the gap

Workaround for issue navigation spacing in Firefox (https://github.com/canonical-web-and-design/vanilla-framework/issues/2483).

Also needed to update `sass-lint` for grid support, so yarn updated a hell of .lock file.

### QA
- ./run or demo
- check navigation in Chrome
- check navigation in Firefox
- they should look the same (no spacing between logo and items in Firefox)
